### ISSUE TITLE
Litmus E2E migration and upgrade meta Issue

### ISSUE THREAD

**Issue by rajdas98** [litmuschaos/litmus#1456]
- Make the litmus e2e scripts to be more generic for adding new cloud provider very easily. - @uditgaurav  @rajdas98  
- Migration of Gitlab runner from GCP VM to local kubernetes cluster @rajdas98  @uditgaurav  Issue: https://github.com/litmuschaos/litmus/issues/1454
- Write terraform scripts to setup EKS on AWS. @rajdas98  Issue: https://github.com/litmuschaos/litmus/issues/1449
- Merge the`generic` branch and `openebs` branch into `master`. As we are migrating gitlab runner to local cluster with Kubernetes executor, we can run the pipeline parallelly. @uditgaurav 

These issues need to be resolve in chronological order.

**Comment by uditgaurav** [comment]
One branch can not have multiple `gitlab-ci.yml`. So even though we have multiple runners we can not combine them in the same branch unit or unless we bundle them (openebs & generic ) in the same `gitlab-ci.yml` with some logic and if we do so we can not run them parallelly so this may be of lesser priority right now. 
I Would like to work on gitlab-runner setup till then...

**Comment by rajdas98** [comment]
@uditgaurav We can achieve this by adding a conditional logic in .gitlabci yaml. The env will be passed while running the pipeline. That's what I created the first issue to make the litmus e2e scripts more generic.

**Comment by ksatchit** [comment]
Could we map/tag them to different runners too ?

**Comment by rajdas98** [comment]
Yes @ksatchit we will use this approach to run the pipeline parallely by taging the runners

**Comment by uditgaurav** [comment]
The target cluster is set up in an on-prem machine for running e2e pipelines.
PR for ref.
generic experiment pipeline - litmuschaos/litmus-e2e#137
openebs experiment pipeline - litmuschaos/litmus-e2e#138
